packages/ag-charts-community/src/chart/marker/marker.ts

```diff
@@ -6,6 +6,20 @@ import { Rotatable, Scalable, Translatable } from '../../scene/transformable';
 
 export type MarkerPathMove = { x: number; y: number; t?: 'move' };
 
+const BUILTIN_MARKERS: readonly string[] = [
+    'ArrowDown',
```

Comment:
Minor - but couldn't we store the actual constructors and do referential equality so it's impossible to fool the check?

Reply:
Technically the base class shouldn't know anything about the derived classes, so this code is sort of hacky anyway. It might be a better idea to create a `BuiltinMarker` subclass of `Marker` which our markers can extend.

Reply:
I've tried improving this logic, but it causes failures in the unit/e2e tests because the `BBox` computation break. In the interest of not diverting from `latest` too much, let's keep this as-is for now.